### PR TITLE
Update ProductListPage to be responsive

### PR DESCRIPTION
**Pull request by MrGreensushi**
Updated Product list page to be responsive.
Also, I deleted the component RedButton: now all buttons are rendered thanks to UI-Components/Button

### CODE REVIEW

File: client/src/ui-components/Title.css [hunk 1]
```diff
@@ -12,7 +12,7 @@
   position: sticky;
   padding: 2% !important;
   padding-bottom: 0% !important;
-  top: 10%;
+  top: 78.3px !important;
```
Comment (MarcoBramini) [inline client/src/ui-components/Title.css]: Setting pixels sizes makes the page not responsive, be careful doing this. Why is this needed?

Comment (MarcoBramini) [reply client/src/ui-components/Title.css]: In general, this css class should not exists. It should be a component named Title

Comment (MrGreensushi) [reply client/src/ui-components/Title.css:15]: I used pixel since I want the Row to be exactly at 78.3 px since it's the height of the navbar, using percentage doesn't work (the height of the page may be different based on the size)

**Review by G-Salviati — APPROVED** [review]

**Review by G-Salviati — APPROVED** [review]
